.gitmodules

```diff
@@ -30,3 +27,6 @@
 [submodule "hybris/selinux_stubs"]
 	path = hybris/selinux_stubs
 	url = https://github.com/mer-hybris/android_external_selinux_stubs
+[submodule "dhd"]
+	path = dhd
+	url = https://github.com/mer-hybris/droid-hal-device
```

Comment:
nit: maybe you could move this block back up for a cleaner diff?

Reply:
ok, will do